deploy/samples/multi-ns-stack.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: monitoring.rhobs/v1alpha1
+kind: MonitoringStack
+metadata:
+  name: multi-ns
+  labels:
+    thanos-querier: the-querier
+spec:
+  logLevel: debug
+  retention: 2h
+
```

Comment:
nit: another white space

Reply:
I have removed it since I need to fix the default sample ...